docs/execute-notebooks.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Execute Notebooks with Jupyter
+title: Execute Notebooks During Your Build
 description: MyST can execute notebooks using Jupyter Server, making it possible to build rich websites and documents from text-based notebooks.
-short_title: Executing Notebooks
+short_title: Execute Notebooks During Your Build
 thumbnail: thumbnails/execute-notebooks.png
 ---
 
-## Overview
-
 :::{warning} MyST Execution is in Beta
-We are adding support for executing markdown notebooks and ipynb files, including inline execution. As we are adding this functionality we appreciate any feedback from the community on how it is working in your environments. Please add [issues](https://github.com/executablebooks/mystmd/issues/new) or join [Discord](https://discord.mystmd.org/) to give feedback.
+By default, execution is disabled and code outputs are only inserted if the notebook has already been executed (for text-based notebooks, there are no outputs).
+We are adding support for executing markdown notebooks and ipynb files, including inline execution.
+As we are adding this functionality we appreciate any feedback from the community on how it is working in your environments. Please add [issues](https://github.com/executablebooks/MyST/issues/new) or join [Discord](https://discord.MyST.org/) to give feedback.
```

Comment:
```suggestion
As we are adding this functionality we appreciate any feedback from the community on how it is working in your environments. Please add [issues](https://github.com/executablebooks/mystmd/issues/new) or join [Discord](https://discord.mystmd.org/) to give feedback.
```

Reply:
ahhh that explains why I got diff conflicts haha, sorry about that I didn't notice the change